### PR TITLE
update rest of seqkit modules to 2.8.1

### PR DESCRIPTION
**Pull request by mashehu**
remove unnecessary  `{ assert snapshot(process.out.versions).match("versions") }` when there is already `{ assert snapshot(process.out).match() }`

### CODE REVIEW

**Comment by SPPearce** [comment]
Are you planning on updating the remaining pytests to nf-test too?

**Comment by mashehu** [comment]
ran out of time...

**Comment by maxulysse** [comment]
Why closing it?

File: modules/nf-core/seqkit/grep/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    test("with_file") {
+
+        when {
+            process {
+                """
+                input[0] = [[ id:'test', single_end:false ], // meta map
+                            file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
+                            ]
+                input[1] = file(params.test_data['homo_sapiens']['genome']['genome_header'], checkIfExists: true)
```
Comment (maxulysse) [inline modules/nf-core/seqkit/grep/tests/main.nf.test:20]: Can you update the paths?

Comment (maxulysse) [reply modules/nf-core/seqkit/grep/tests/main.nf.test:20]: Good enough

File: modules/nf-core/seqkit/seq/tests/main.nf.test [hunk 2]
```diff
@@ -26,7 +26,6 @@ nextflow_process {
             assertAll(
                 { assert process.success },
                 { assert snapshot(process.out).match() },
-                { assert snapshot(process.out.versions).match("versions") }
```
Comment (heuermh) [inline modules/nf-core/seqkit/seq/tests/main.nf.test:29]: Thanks!  This is what I was hoping for, no longer considering checksums on the versions file.

Comment (SPPearce) [reply modules/nf-core/seqkit/seq/tests/main.nf.test:29]: Well, the versions is being checked in the line above, as it is part of `process.out`

**Review by maxulysse — APPROVED** [review]

**Review by heuermh — APPROVED** [review]
Thanks!

**Review by maxulysse — APPROVED** [review]

**Pull request merged**